cropro.py

```diff
@@ -339,6 +377,8 @@ def reject(self):
 def addMenuItem():
     a = QAction(mw)
     a.setText('Cross Profile Search and Import')
```

Comment:
I do not know about this project, but please double-check if any kind of localization is in place. If so - these strings should be replaced with appropriate implementation.